Guides/Migrating.md

```diff
@@ -40,7 +40,15 @@ Also update dependencies version to latest if required.
 
 And run pod install. If successful, running and testing will work correctly. 
 
-#### 9. In example project, add the newly added destination by setting its relative path.
+#### 9. For tests to compile successfully from command line, code signing for the test targets must be set. (It's enough that it's a personal team, Apple ID does not need to have purchased developer program)
+
+#### 10. It may happen that the pod install has created extra schemes in workspace, simply delete them, but (choose Remove reference) upon delete confirmation.
```

Comment:
Can you provide more detail for users unrelated to Xcode projects where they are supposed to be findnig/deleting these extra schems in the workspace. I think if we can make these steps as explicit as possible the better. Ie. Steps like go here in Xcode, Produc  -> Scheme -> Edit Scheme and then explicitly state which "extra schemes" the user should be removing or should deem as extra.